README.md

```diff
@@ -15,7 +15,7 @@
   </a>
 </p>
 
-> Vocabulary is a cohesive design system to unite the web facing Creative 
```

Comment:
These changes only serve to increase the PR size without adding value to the project. Please undo such changes.